src/commands/subcommands/github/handleAddLabels.ts

```diff
@@ -18,18 +18,38 @@ export const handleAddLabels: Subcommand = {
       const labels = interaction.options.getString("labels", true);
       const labelNames = labels.split(",").map((l) => l.trim());
 
-      await Bot.octokit.issues.addLabels({
-        owner: "freeCodeCamp",
-        issue_number: number,
-        repo,
-        labels: labelNames
-      });
+      const response = await Bot.octokit.request(
+        "GET /repos/{owner}/{repo}/labels",
+        {
+          owner: "freeCodeCamp",
+          repo: repo
+        }
+      );
+
+      const presentLabels = response.data.map((x) => x.name.trim());
+
+      if (
+        labelNames.every((requestedLabel) =>
+          presentLabels.includes(requestedLabel)
+        )
```

Comment:
We should invert the logic. And leverage early return patterns.

if (label not valid) {
  interaction.editReply(label don't exist)
  return
}

add label
interaction.editReply(label added)

Reply:
Understood.